src/nifgen/examples/nifgen_trigger_example.py

```diff
@@ -0,0 +1,54 @@
+import argparse
+import nifgen
+import sys
+import time
+
+
```

Comment:
[Run the code generator](https://github.com/ni/nimi-python/blob/master/CONTRIBUTING.md#how-to-build) with your changes. That will update this file: https://github.com/ni/nimi-python/blob/master/docs/nifgen/examples.rst

Include the updated file in the PR.